packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -223,12 +224,73 @@ void updateLocalProperties({String projectPath, BuildInfo buildInfo}) {
     settings.writeContents(localProperties);
 }
 
+void findAndReplaceVersionProperties({BuildInfo buildInfo}) {
```

Comment:
Looks like `buildInfo` can't be null, so make this `@required`, and assert that it's non-null at the start of the function.